### PR TITLE
Started the project and modified the README and .gitignore

### PR DESCRIPTION
**Pull request by tkz96**
Started the development of my personal portfolio website and created the welcome card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>    
+    
+    <header>
+        <nav class="app-header">
+        <p class="logo-placeholder">Welcome!</p>
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] As you can see clearly in almost websites the `logo` should be a button to navigate to the home page (index.html), Kindly wrap your logo in an  anchor element (`a` tag)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                <img src="menu-button.svg" alt="Navigation">
+        </nav>
+    </header>
+    <div class="app-wrapper">
+        
+        <section class="welcome-line">    
+            <h2>I am Talha.<br>Glad to see you!</h2>
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] Kindly use the `h1` tag here, that will make your code more semantic and structured

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @tkz96, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @tkz96, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions
- Please give the branch a meaningful name like the name of the project `setup-and-mobile-skeleton`, not `project-build-1`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tkz96** [comment]
Understood. I will take care of naming the branches.

**Pull request merged**